start.bat

```diff
@@ -1,2 +1,3 @@
+@echo off
 node . --trace-warnings
 PAUSE
```

Comment:
It’s customary to end source files with a newline.

Have a look at [^1] for reasons.

[^1]: https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file